smsdk/_version.py

```diff
@@ -0,0 +1,113 @@
+#
+"""
+Track version information for sightmachine-sdk module here.
+"""
+import typing
+import itertools
+import requests
+import warnings
+import functools
+import re
+
+
+class VersionInfo(typing.NamedTuple):
+    major: typing.Any
+    minor: typing.Any
+    patchlevel: typing.Any
```

Comment:
Should these be of type `int`? FYI, I believe a default value can be added to fields as well. So `serial` can default to `None`.

Reply:
Corrected these static types.